spec/models/request/sample_compound_aliquot_transfer_spec.rb

```diff
@@ -51,7 +51,7 @@
     end
   end
 
-  describe '#transfer_aliquots_into_compound_sample_aliquot' do
+  describe '#transfer_aliquots_into_compound_sample_aliquots' do
```

Comment:
Does this test need anything adding to represent the creation of multiple compound sample aliquots

Reply:
I think that's probably covered in `sequencing_request_spec` by 'when there are two tag combinations'? Don't want to create too much duplication